README.md

```diff
@@ -5,6 +5,14 @@
 - Node: Version 16.13.0
 - Npm: Version 8.1.0
 
+## Environment
+
+To define environment variables we are using: https://create-react-app.dev/docs/adding-custom-environment-variables/. The available variables are:
+
+**REACT_APP_TERRASO_API_URL**: Terraso API host URL
+
+### Set variable: `export REACT_APP_TERRASO_API_URL=http://localhost:8000/graphql/`
```

Comment:
When this variable is used? During the build time? If so, perhaps it's worth also inform some way to run the build command with proper configuration. Only export the environment variable might lead to misunderstanding. So, if the environment variable is used on build, we could add the following to the docs:
```sh
# To build the application with custom API URL
REACT_APP_TERRASO_API=https://terraso.com/graphql npm run build
```

Reply:
@cacarrara These variables are used at build time and when you start it too, so it will be for both:
`REACT_APP_TERRASO_API=https://terraso.com/graphql npm run build`
`REACT_APP_TERRASO_API=https://terraso.com/graphql npm start`

I will add this